src/index.html

```diff
@@ -16,6 +15,34 @@
      Description Language (WDL), or Nextflow (NFL)"
     />
 
+    <!-- Style sheet from https://fonts.googleapis.com/icon?family=Material+Icons&display=swap  -->
+    <!-- Google fonts does not support SRI (https://github.com/google/fonts/issues/473) -->
+    <style>
+      @font-face {
+        font-family: 'Material Icons';
+        font-style: normal;
+        font-weight: 400;
+        font-display: swap;
+        src: url('assets/fonts/MaterialIcons.woff2') format('woff2');
```

Comment:
I am concerned this may not work in a deployed environment. You may need to precede it with `..`. See other examples of `src: url`. The reason is [here](https://github.com/dockstore/dockstore-ui2/blob/0f3a414bd285649c9bbbc9211b2b948a8bac2d84/.circleci/config.yml#L174) -- when we build the UI for deployment, we replace `../assets` with `https://gui.dockstore.org/....`. This is for the CloudFront deploy that I touched upon the other day in storytime.

Reply:
I've been looking into other approaches, per Gary's comments above. The idea was to see if importing from the url was handled at build or runtime, and if we could potentially leverage this to avoid SRI issues. From what I've been able to determine, importing from a remote url is resolved at runtime.

It looks like Google has a node module of the material icons, https://www.npmjs.com/package/material-design-icons , which we could potentially switch to?

Reply:
It looks like the WOFF file they generate gets some unpredictable URL https://fonts.gstatic.com/s/materialicons/v55/flUhRq6tzZclQEJ-Vdg-IuiaDsNc.woff2 . It doesn't seem like a good idea to include that URL instead in our assets, but worth considering.

Fonts and icons are a big download and having them downloaded from CDN can have a noticeable impact on experience.

Regarding the problem of not knowing the base URL, maybe we could use a directory called "fonts" that is managed separately from releases? This would allow us to get caching behavior across releases and have good guarantees for the base URL. Or, we could just put the fonts in develop and link to that build, specifically, here.

Reply:
After reading through, https://www.tunetheweb.com/blog/should-you-self-host-google-fonts/ (interesting read!) there doesn't seem to be a clear winner between self-host and using the google web-fonts, even though the author concludes with recommending self-hosting.

If we were to follow the self-hosting approach, I tested the node-module release of the material icons and it seems well-rounded (better than the current implementation in the PR). It supplies multiple file types (eot, ttf, woff, woff2) bundled in the build for browser compatibility, of which the browser only needs to load 1.

Referencing David's suggestion above, we could include the multiple file types seen in the node module in a separate "fonts" directory, to get a little more breadth to the caching potential?